src/live.rs

```diff
@@ -343,6 +349,48 @@ impl<'a> KargEmbedAreas<'a> {
         get_kargs_at_offset(self.file, self.length, self.default_kargs_offset)
     }
 
+    fn stream(&mut self, kargs: &str, writer: &mut (impl Write + ?Sized)) -> Result<()> {
+        let mut buf = [0u8; BUFFER_SIZE];
+        let kargs: String = kargs.trim().to_string() + "\n";
+        if kargs.len() > self.length {
+            bail!(
+                "kargs too large for area: {} vs {}",
+                kargs.len(),
+                self.length
+            );
+        }
+
+        self.file
+            .seek(SeekFrom::Start(0))
+            .context("seeking to start")?;
+        let mut new_area = vec![b'#'; self.length];
+        new_area[..kargs.len()].copy_from_slice(kargs.as_bytes());
```

Comment:
There's enough duplicated code setting up `new_area` that it should probably just move to a helper method.

Reply:
I guessed a name b/c `let new_area = self.new_area(...)` looked silly. Let me know what you think.

Reply:
Hmm, it's a bit generic.  Something like `format_embed_area`?